ArdusatSDK.cpp

```diff
@@ -1,35 +1,52 @@
 /**
  * @file   ArdusatSDK.cpp
- * @Author Ben Peters (ben@ardusat.com)
+ * @author Ben Peters (ben@ardusat.com)
+ * @author Sam Olds (sam@ardusat.com)
  * @date   December 3, 2014
  * @brief  Implements ArdusatSDK generic sensor reading and configuration for Space Kit Sensors.
+ *
+ * Provides a unifying wrapper of sensor specific functionality to provide a
+ * consistent interface to interact with each type of sensor.
  */
 
 #include <stdio.h>
 #include <string.h>
 #include "ArdusatSDK.h"
 
-bool ARDUSAT_SHIELD = false;
-int OUTPUT_BUF_SIZE = 256;
+boolean MANUAL_CONFIG = false;
+boolean ARDUSAT_SPACEBOARD = false;
+int OUTPUT_BUF_SIZE = 512;
```

Comment:
Having only 323 bytes available for RAM worries me. That's relatively easy to overflow the stack in, especially in a complicated function or deep call stack...

In all sensors, how many bytes long is the first row?

It's a little hacky, but we might want to just pass a reference to the serial object into the write header function of the CSV writer, rather than outputting a `char` pointer, and then do the writes themselves inside the function so that we can split the buffer write into 2 operations but not insert a newline...